chalk-parse/src/ast.rs

```diff
@@ -24,6 +24,8 @@ pub enum Item {
     TraitDefn(TraitDefn),
     Impl(Impl),
     Clause(Clause),
+    Scalar(Scalar),
+    // TODO: tuple
```

Comment:
I don't think we need these here -- there are no "items" corresponding to scales and tuples. Items are declarations like `struct Foo { .. }`